### PR TITLE
Support for header bar

### PR DESCRIPTION
**Pull request by skyronic**
Support for a basic header bar with a svg logo and a box-shadow based border according to spec.

image

### CODE REVIEW

File: src/components/HeaderBar.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  )
+}
+
+class HeaderBar extends PureComponent<Props> {
+  static defaultProps = {}
+
+  render () {
+    return (
+      <div className={cx.logoWrapper}>
+        <div className={cx.logoContainer}>{svgLogo(84, 30)}</div>
```
Comment (bernardodiasc) [inline src/components/HeaderBar.js]: Hi @skyronic, is there any reason to use `<div className={cx.logoContainer}>`? I notice the `logoContainer` don't have any styles.

File: src/components/HeaderBar.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+const cx = {
+  logoWrapper: cmz(`
+    display: flex;
+    justify-content: center;
+    box-shadow: 0 5px 5px -5px rgba(0,0,0,0.25);
+    height: 58px;
+    padding-top: 28px;
+    width: 100%;
+`),
+  logoContainer: cmz(`
```
Comment (nicksp) [inline src/components/HeaderBar.js]: Shall we remove an empty style here?

**Comment by skyronic** [comment]
@bernardodiasc @nicksp yeah you're right. Sorry I was trying to mess with the CSS to do centering and forgot to remove the obsolete class. Will update PR

**Comment by joshwnj** [comment]
@skyronic looks good. I've removed that empty class and the wrapper div.

**Pull request merged**